### DISCUSSION TITLE
Turn display off after x minutes of inactivity?

### DISCUSSION THREAD

**Discussion by tgreening** [sethvoltz/OctoPrint-DisplayPanel#53] [Ideas]
I was thinking that eventually the lcd will burn out due to constantly being on.  Maybe turn off the display after x minutes of inactivity to save the lcd?  I know the PSU control does this so maybe they have the code.

Love this BTW make controlling my printer much easier.

**Comment by Andy-ABTec** [comment]
I think what you're looking for is already there. In Settings > OctoPrint Micro Panel; look on the **SECOND** tab towards the bottom of the form - "Blank the Display when..." :sunglasses:

![image](https://user-images.githubusercontent.com/5175695/107153498-c7d26580-6965-11eb-9126-9cfc53f626de.png)

**Comment by sethvoltz** [comment]
@tgreening does that fit your needs?

**Reply by tgreening** [reply]
absolutely!  I feel stupid that I did not see this feature!

Thanks and great job!
